pages/version-policy/index.md

```diff
@@ -121,24 +109,6 @@ The following matrix shows the platform components and operating environments on
         <td><p style="text-align: center;"><ul><li>Docker CE 18.06.1</li></ul></p></td>
         <td><p style="text-align: center;"><ul><li>Docker CE 18.06.1</li></ul></p></td>
     </tr>
-    <tr>
-        <td>CoreOS 1911.3.0</td>
-        <td><p style="text-align: center;"><ul><li>Docker CE 18.06.1</li></ul></p></td>
-        <td><p style="text-align: center;"><ul><li>Docker CE 18.06.1</li></ul></p></td>
-        <td>                    </td>
-    </tr>
-    <tr>
-        <td>CoreOS 1855.5.0</td>
-        <td><p style="text-align: center;"><ul><li>Docker CE 18.06.1</li></ul></p></td>
-        <td><p style="text-align: center;"><ul><li>Docker CE 18.06.1</li></ul></p></td>
-        <td>                    </td>
-    </tr>
-    <tr>
-        <td>CoreOS 1855.4.0</td>
-        <td><p style="text-align: center;"><ul><li>Docker CE 18.06.1</li></ul></p></td>
-        <td><p style="text-align: center;"><ul><li>Docker CE 18.06.1</li></ul></p></td>
-        <td>                    </td>
-    </tr>
     <tr>
         <tr>
         <td>RHEL 7.6</td>
```

Comment:
RHEL is in there twice. Can you please remove the second instance of RHEL? Starts on line 114.